source/indexes/create-search-index.txt

```diff
@@ -0,0 +1,89 @@
+.. _compass-create-search-index:
+
+============================
+Create an Atlas Search Index
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. versionadded:: 1.40.0
+
+You can create Atlas Search indexes in |compass|. Atlas Search indexes
+let you query data in `Atlas Search
+<https://www.mongodb.com/docs/atlas/atlas-search/>`__. Atlas Search
+indexes enable performant text search queries by mapping search terms to
+the documents that contain those terms.
+
+About this Task
+---------------
+
+To create an Atlas Search index must meet the following requirements:
+
+- Hosted on `MongoDB Atlas <https://www.mongodb.com/docs/atlas/>`__
```

Comment:
We can also create Atlas Search indexes on local clusters set up using the Atlas CLI: https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-deploy-local/

Reply:
Thanks! I'll rework this section a bit.